### PR TITLE
Add support for mod_auth_cas module configuration

### PR DESCRIPTION
**Pull request by pcfens**
Add support to configure mod_auth_cas within Apache.

### CODE REVIEW

File: manifests/mod/auth_cas.pp [hunk 1]
```diff
@@ -0,0 +1,48 @@
+class apache::mod::auth_cas (
+  $cas_login_url            = undef,
+  $cas_validate_url         = undef,
```
Comment (mhaskel) [inline manifests/mod/auth_cas.pp]: Since these two variables are required, they should not be set to undef but rather have no default value.

**Comment by mhaskel** [comment]
@pcfens thanks for the contribution, and sorry it took so long for us to get back to this. I think this is pretty close to being ready to merge. I made one inline comment that needs to be addressed, and it looks like tests are failing against 1.8.7, but that's possibly been fixed upstream, so if you rebase against master we'll have a better idea. thanks!

**Comment by pcfens** [comment]
@mhaskel Thanks for the feedback - I've made the suggested changes and fixed the test issues as well.

Thanks again.

**Comment by mhaskel** [comment]
Looks good, thanks @pcfens !

**Pull request merged**